.buildkite/scripts/build_component.sh

```diff
@@ -84,15 +84,28 @@ echo "--- :habicat: Uploading ${pkg_ident} to Builder in the '${channel}' channe
 #     --auth="${HAB_AUTH_TOKEN}" \
 #     "${pkg_ident}" "${channel}" "${pkg_target}"
 
-curl -v -X POST \
-    -H "Accept: application/json" \
-    -H "Content-Type: application/json" \
-    -H "Authorization: Bearer $HAB_AUTH_TOKEN" \
-    --data-binary "@results/${pkg_artifact}.hart" \
+curl --request POST \
+     --header "Content-Type: application/octet-stream" \
+     --header "Authorization: Bearer $HAB_AUTH_TOKEN" \
+     --data-binary "@results/${pkg_artifact}" \
+     --location \
+     --post301 \
+     --fail \
+     --verbose \
     "http://bldr.habitat.sh/v1/depot/pkgs/${pkg_ident}?checksum=${pkg_blake2bsum:?}&target=${pkg_target}"
```

Comment:
why not just convert this http to https and avoid the redirect?

Reply:
Derp.